api/views.py

```diff
@@ -160,8 +164,11 @@ def sendText(numbers):
                 body= f'\nToday\'s NASA Astronomy Picture of the Day is: {data["title"]}.\n\n{data["explanation"]}'
                 )
                 print(message.sid)
+
+            return True
```

Comment:
you are returning within a loop. This will mean your function will return before getting through all the `numbers`

I suggest having a function for a single number `sendText(number`) and having `sendApod` iterate over the numbers

e.g.
```python
for number in getNumbers():
     did_send = sendText(number)```

Reply:
Should 'did_send be a block variable or a function variable so I can print the right output

Reply:
nevermind